cmake/LYTestWrappers.cmake

```diff
@@ -313,12 +313,14 @@ function(ly_add_pytest)
     # Add the script path to the test target params
     set(LY_TEST_PARAMS "${ly_add_pytest_PATH}")
 
+    string(REPLACE "::" "_" pytest_output_directory "${CMAKE_BINARY_DIR}/Testing/LyTestTools/${ly_add_pytest_NAME}")
```

Comment:
suggest to set the first part (without the test) to a variable up there on line 33-34 so its together with the other ones
```
ly_set(LYTESTTOOLS_OUTPUT_DIR "${CMAKE_BINARY_DIR}/Testing/LyTestTools")
```

Reply:
fixed